.ci/docker/common/install_inductor_benchmark_deps.sh

```diff
@@ -7,18 +7,14 @@ source "$(dirname "${BASH_SOURCE[0]}")/common_utils.sh"
 function install_huggingface() {
   local version
   version=$(get_pinned_commit huggingface)
-  pip_install pandas
-  pip_install scipy
```

Comment:
scipy and z3-solver have already been installed as part of `requirements-ci.txt`